### PR TITLE
Fix bug setting poll and create two link for poll

### PR DESCRIPTION
**Pull request by quanttn-0861**
Peek 2019-04-23 15-34

### CODE REVIEW

File: app/Http/Controllers/LinkController.php [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    public function __construct(LinkRepositoryInterface $linkRepository)
+    {
+        $this->linkRepository = $linkRepository;
+    }
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
```
Comment (framgiaciv3) [inline app/Http/Controllers/LinkController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/LinkController.php</br>**Line**: 21</br>Variable "link_user" is not in valid camel caps format

File: app/Http/Controllers/LinkController.php [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    {
```
Comment (framgiaciv3) [inline app/Http/Controllers/LinkController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/LinkController.php</br>**Line**: 22</br>Variable "link_admin" is not in valid camel caps format

File: app/Http/Controllers/LinkController.php [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        $link_user = $this->linkRepository->createLinkUser();
+        $link_admin = $this->linkRepository->createLinkAdmin();
```
Comment (framgiaciv3) [inline app/Http/Controllers/LinkController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/LinkController.php</br>**Line**: 24</br>Variable "link_user" is not in valid camel caps format

Comment (framgiaciv3) [inline app/Http/Controllers/LinkController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/LinkController.php</br>**Line**: 24</br>Variable "link_admin" is not in valid camel caps format

File: app/Http/Controllers/LinkController.php (hunk 1 above)
Comment (framgiaciv3) [inline app/Http/Controllers/LinkController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/LinkController.php</br>**Line**: 21</br>Variable "link_user" is not in valid camel caps format

File: app/Http/Controllers/LinkController.php (hunk 2 above)
Comment (framgiaciv3) [inline app/Http/Controllers/LinkController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/LinkController.php</br>**Line**: 22</br>Variable "link_admin" is not in valid camel caps format

File: app/Http/Controllers/LinkController.php (hunk 3 above)
Comment (framgiaciv3) [inline app/Http/Controllers/LinkController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/LinkController.php</br>**Line**: 24</br>Variable "link_user" is not in valid camel caps format

Comment (framgiaciv3) [inline app/Http/Controllers/LinkController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/LinkController.php</br>**Line**: 24</br>Variable "link_admin" is not in valid camel caps format

File: resources/js/components/poll/LinkPoll.js [hunk 4]
```diff
@@ -4,9 +4,27 @@ import { Link } from 'react-router-dom';
...
+            link_user: link_user,
+            link_admin: link_admin,
+        })
+    }
+
     render() {
-        var link_user = "link here"
-        var link_admin = "link here"
+        var link_user = "http://127.0.0.1:8000/vote/"+this.state.link_user
+        var link_admin = "http://127.0.0.1:8000/vote/"+this.state.link_admin
```
Comment (tranducquoc) [inline resources/js/components/poll/LinkPoll.js]: sao lại để link như này? lên các môi trường khác thì thế nào?

Comment (quanttn-0861) [reply resources/js/components/poll/LinkPoll.js]: Em đã đổi lại thành dùng window.Laravel.baseUrl, biến javascript window.Laravel sẽ chứa những giá trị động lấy từ Laravel sử dụng trong javascript.

**Pull request merged**